grinder/interface.py

```diff
@@ -83,12 +83,24 @@ def parse_args(self) -> Namespace:
             default="queries.json",
             help="JSON File with Shodan queries",
         )
+        parser.add_argument(
+            "-target",
+            "--host-ip",
```

Comment:
Why the short form is `target`, but the full form is `--host-ip`? It's not logical.